src/libraries/Microsoft.PowerFx.Core/Types/DType.cs

```diff
@@ -578,6 +578,8 @@ internal void AssertValid()
 
         public bool IsTable => Kind == DKind.Table || Kind == DKind.ObjNull || Kind == DKind.LazyTable;
 
+        public bool IsTableStrict => Kind == DKind.Table || Kind == DKind.LazyTable;
```

Comment:
In the future, we _might_ have to add a Boolean flag with Kind == DKind.Table || Kind == DKind.ObjNull without the LazyTable one. Then, isn't that one also "Strict?" We are currently not using the word "Strict" anywhere in this document. Could we make it more descriptive? Maybe IsNonBlankTable?

Reply:
Updated it to `IsTableNonObjNull`